docker/artemis-migration-check-postgres.yml

```diff
@@ -0,0 +1,38 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Artemis-Migration-Check-Postgres Setup
+# ----------------------------------------------------------------------------------------------------------------------
+
+services:
+    artemis-app:
+        extends:
+            file: ./artemis.yml
+            service: artemis-app
+        env_file:
+            - ./artemis/config/cypress-postgres.env
```

Comment:
Is this here on purpose?

Reply:
Yes it is needed, so artemis uses the postgres connector instead of the mysql connector. I can rename the file to something more generic (maybe `postgres-dev.env` or something)

Reply:
@maximiliansoelch, @TheZoker this is already handled [in this PR](https://github.com/ls1intum/Artemis/pull/6759) where I also add the development postgres setups ;)